server/py/services/api/tests/unit/db/test_artifacts.py

```diff
@@ -737,12 +737,12 @@ def test_modify_artifact_tags(self):
             self._db_session, project=project, name=artifact_key
         )
         assert len(artifacts) == 3
+        assert artifacts[0]["metadata"]["tag"] == "v2"
```

Comment:
please link the follow up PR in the description because this looks weird out of context